docs/wallet/add-edit-wallet-address.mdx

```diff
@@ -74,7 +74,7 @@ To edit or change your wallet address, go to settings screen and tap "My wallet"
 ## How to create wallet on Metamask
 
 While we are on the testnet, we suggest you use a **MetaMask** wallet with WeatherXM. By following our guide, it's
```

Comment:
This is wrong. Replace this. We need to write something that will say that we have tested Metamask etc. and in the future we will add a paragraph showing the supported wallets.